l10n_ir_account/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Iran - Accounting",
+    "version": "1.0.0",
```

Comment:
```suggestion
    "version": "17.0.1.0.0",
```

@saeed-raeisi @dreispt this is the cause of the merge error. We'll need to investigate why pylint-odoo did not flag this error, and why the bot did not report the exception on the PR.